wiki/en/Server-Troubleshooting.md

```diff
@@ -19,17 +19,17 @@ permalink: "/wiki/Server-Troubleshooting"
 
 </details>
 
-## Servers - Public
+## Servers - Registered 
 
 ### Why doesn't my server show up in the list? Why isn't it registering?
 
 If you are registered OK (you can [see it here](https://explorer.jamulus.io/)) and you or your friends can't see your server, you may need to wait, or start your client with the `--showallservers` option and try connecting from there ([see command line options](Software-Manual#command-line-options)) on how to start your client with a config option).
```

Comment:
It's worth mentioning again that the magic port forwarding fix may not be working.